bin/test-backend.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+NAMESPACE=staging
+IMAGE_TAG=healthcheck2
+INGRESS_STATIC_IP_NAME=travella-staging-frontend
```

Comment:
I think this should be `travella-staging-backend`, yes?

Reply:
No, both are sharing the same IP address

Reply:
Okay... Then should it have a more generic name? Like `travella-staging` or something

Reply:
Yes, it should be. But unfortunately, GCP does not provide us with the functionality to rename the IP, hence we would need to create a new one, which for now would break the application and there will be a need to update DNS, which I do not have the permission to do so.

Reply:
Ow.. Noted with thanks. Everything else seems okay on my part